### DISCUSSION TITLE
Page with Tailwind CSS returns error on Google Pagespeed

### DISCUSSION THREAD

**Discussion by nsursock** [tailwindlabs/tailwindcss#1912] [Help]
Have a look: https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fsoundsirius.digitalpress.blog%2Fproducts%2F&tab=mobile
Also it seems it takes more than 1 second for the framework to load.
I need a fast website but i really want to use tailwind.

**Comment by estevanmaito** [comment]
Your page takes 4.6s to load, and images take 3.29s to load, yet you think that the performance problem is the 0.6s that CSS takes?

Anyways, you're using the CDN version of Tailwind (it takes 1s just to connect to the host, compared to your self-hosted `app.css`), and could maybe bring that 0.6s down to 0.2s if you purge https://tailwindcss.com/docs/controlling-file-size

**Reply by TrentFoulds** [reply]
Just adding height and width attributes and classes won't work, you have to crop the images as well to its exact dimension. For example if you have set 100px height and width, the image height and width should also be 100px.  [FMC4ME Application](https://www.fmc4me.net/)

**Comment by kmfusa9-afk** [comment]
Yes—Tailwind CSS itself usually isn’t the problem. If Google PageSpeed is reporting an error/warning on a Tailwind page, the exact audit matters. [fmc4me sign on login](https://fmc4me.cloud/)
